lints/base.go

```diff
@@ -2,12 +2,14 @@ package lints
 
 import (
 	"github.com/zmap/zgrab/ztools/x509"
+	"math/big"
 	"time"
 )
 
 // global
 var (
-	Lints map[string]*Lint = make(map[string]*Lint)
+	Lints      map[string]*Lint = make(map[string]*Lint)
+	upperBound                  = &big.Int{}
```

Comment:
I'm really not sure why this is global across the entire zlint. Why can't this be stored on the rsaParsedTestsExpInRange object? AFAICT that is the only place this is used. Each lint has its own Initialization function.